INSTALL.md

```diff
@@ -214,6 +215,16 @@ to be missing, please open an issue.
 swupd bundle-add devpkg-expat devpkg-freetype devpkg-libxcb devpkg-fontconfig
 ```
 
+#### Alpine Linux
+
+On Alpine Linux, you need a few extra libraries to build Alacritty. Here's an
+`apk` command that should install all of them. If something is still found to
+be missing, please open an issue.
+
+```sh
+sudo apk add cmake pkgconf fontconfig-dev python3 libxcb-dev
```

Comment:
You need freetype here as well.

Reply:
I'd missed this because `fontconfig-dev` has `freetype-dev` as a dependency. However, in hindsight, it also has `pkgconf` as a dependency---now we should be consistent with all the dependencies explicitly listed.